rasa/server.py

```diff
@@ -889,3 +903,32 @@ def validate_request(rjs):
             )
 
     return app
+
+
+def _get_output_channel(
```

Comment:
Can you add some comments inside the method? It is quite hard to understand what exactly is happening.

Reply:
🙈